src/core/Microsoft.PowerToys.Settings.UI/Views/ColorPickerPage.xaml

```diff
@@ -68,7 +68,10 @@
                           Width="240"
                           IsEnabled="{Binding IsEnabled}">
                 <ComboBoxItem Content="HEX - #FFAA00"/>
-                <ComboBoxItem Content="RGB - RGB(100, 50, 75)"/>
+                <ComboBoxItem Content="RGB - rgb(100, 50, 75)"/>
+                <ComboBoxItem Content="CYMK - cymk(100%, 50%, 75%, 0%)"/>
+                <ComboBoxItem Content="HSL - hsl(100, 50%, 75%)"/>
+                <ComboBoxItem Content="HSV - hsv(100, 50%, 75%)"/>
```

Comment:
Because the first number define the degree, I think we could use the `°` sign here.

Reply:
Hello, my intention was to reflect the clipboard format
When use the HSL value in CSS you can't write/use a degree sign
```css
/* error */
hsl(100°, 50%, 75%)

/* working */
hsl(100, 50%, 75%)
```

Reply:
> 
> 
> @TobiasSekan
> You write in the PR description that you have the problem that the ui is to small. I think @martinchrzan could help you with this problem.

Yes i have try to change the width value from the `<Window>` inside the `ColorPickerUI` Project, but it change the size or it was to big and it have use a fallback value, I don't know.

Reply:
goal here is to be able to do direct pastes, i'm good with no degree symbol